Jenkinsfile

```diff
@@ -98,8 +98,8 @@ pipeline {
                 )
               }
 
-              if (cambpmWithLabels()) {
-                // only execute on `master`
```

Comment:
I wouldn't remove this code. We need to restore it once we put it in production.